docs/openapi.yaml

```diff
@@ -975,6 +1365,190 @@ components:
+          minimum: 1
+          maximum: 31
+          exclusiveMinimum: false
+          exclusiveMaximum: false
+        price:
+          type: integer
+          format: int32
+          example: 7900
+          description: Treated as a whole sum postfixed with two decimals for cents, eg 234 = 2.34 NOK
+        productDescription:
+          type: string
+          example: Fin abonnementspakke
+        productName:
+          type: string
+          example: Pluss-abonnement
+          description: Name of the product being subscribed to.
+        start:
+          type: string
+          example: '2019-01-01T00:00:00Z'
+          description: Date when agreement was started.
+          readOnly: true
+        status:
+          type: string
+          example: ACTIVE
+          description: Status of the agreement.
+          enum:
+            - PENDING
+            - ACTIVE
+            - STOPPED
+            - EXPIRED
+        stop:
+          type: string
+          example: '2019-01-01T00:00:00Z'
+          description: Date when agreement was stopped.
+          readOnly: true
+        merchantAgreementUrl:
+          type: string
+          example: https://www.merchant.no/subscriptions/1234/
+          description: URL where Vipps can redirect the customer to view/administer their subscription.
+        sub:
+          type: string
+          example: 8d7de74e-0243-11eb-adc1-0242ac120002
+          description: User identifier (subject). Will be null if profile data was not requested.
+        userinfoUrl:
+          type: string
+          example: https://api.vipps.no/vipps-userinfo-api/userinfo/8d7de74e-0243-11eb-adc1-0242ac120002
+          description: The full path of the URL for the userinfo endpoint where the user data can be retrieved. Will be null if profile data was not requested.
+        variableAmount:
+                  $ref: "#/components/schemas/VariableAmount"
+      title: Agreement
+
+    PatchAgreementV3:
+      type: object
+      properties:
+        suggestedMaxAmount:
+          type: integer
+          format: int32
+          example: 7900
+          minimum: 100
+          description: Treated as a whole sum postfixed with two decimals for cents, eg 234 = 2.34 NOK
+        productDescription:
+          type: string
+          example: Fin abonnementspakke
+        productName:
+          type: string
+          example: Pluss-abonnement
+          description: Name of the product being subscribed to.
+        merchantAgreementUrl:
+          type: string
+          example: https://www.merchant.no/subscriptions/1234/
+          description: URL where Vipps can redirect the customer to view/administer their subscription.
+        status:
+          type: string
+          example: ACTIVE
+          description: Status of the agreement.
+          enum:
+            - STOPPED
```

Comment:
The enum seems a bit short?

Reply:
The only valid status for a merchant to patch an agreement with is STOPPED, as they cannot do anything else with status